src/Installers/Windows/AspNetCoreModule-Setup/CustomAction/aspnetcoreCA.vcxproj

```diff
@@ -49,7 +49,8 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '17.0' ">v143</PlatformToolset>
```

Comment:
Why exactly `17.0` here instead of any `17.x` version?

Reply:
Sounds like that's by design, though it isn't obvious - could you add a comment saying that `VisualStudioVersion` is always `Major.0` so that future build people don't make unnecessary changes to this?

Reply:
Or make it `Condition="$(VisualStudioVersion.StartsWith('17'))">`

Reply:
> Or make it `Condition="$(VisualStudioVersion.StartsWith('17'))">`

Please do not do this. The current check is fine.